spinach/task.py

```diff
@@ -36,7 +52,7 @@ def task_name(self):
     def __repr__(self):
         return 'Task({}, {}, {}, {}, {})'.format(
             self.func, self.name, self.queue, self.max_retries,
-            self.periodicity
+            self.periodicity, self.max_concurrency,
```

Comment:
Does this work since the string to format has not been changed?

Reply:
Good catch, there's no test for it so I'll add one.